roles/ffmpeg/tasks/ffmpeg.yml

```diff
@@ -1,25 +1,28 @@
 ---
-- name: link media libraries built from source
-  become: yes
-  shell: /usr/sbin/ldconfig
+- include: ldconfig.yml
 
 - name: clone ffmpeg source
-  git: repo=git://source.ffmpeg.org/ffmpeg depth=1 accept_hostkey=yes dest={{ ffmpeg_path }}/ffmpeg force=yes
+  git: repo=https://github.com/FFmpeg/FFmpeg.git \
+       depth=1 dest={{ ffmpeg_path }}/ffmpeg force=yes
 
 - name: configure ffmpeg
-  shell: cd {{ ffmpeg_path }}/ffmpeg && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./configure --extra-libs="-ldl" --enable-gpl --enable-nonfree --enable-libfdk_aac --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libvpx --enable-libx264
+  environment:
```

Comment:
Do you mean it should be
```yaml
environment: PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
```
I think either works, but if you prefer the compact style here I'll change it.